src/content/docs/pt-br/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,612 @@
+
+Como alternativa, delete todas as referências importando o componente `<Markdown />` do Astro e o próprio componente em seus arquivos `.astro`. Você vai precisar escrever novamente seu conteúdo como HTML diretamente ou [importar Markdown](/pt-br/guides/markdown-content/#importando-markdown) de um arquivo `.md`.
+
+### Removido: APIs depreciadas da 1.x
+
+No Astro v1.x, Astro depreciou opções da configuração assim como os tipos de atributo de script/style `global` e `hoist`. Esses foram mantidos por compatibilidade com versões anteriores.
+
+Astro v3.0 remove essas APIs depreciadas completamente. Elas não podem mais ser usadas no seu projeto Astro.
+
+#### O que devo fazer?
+
+Se você continua utilizando APIs da v1.x, utilize as novas APIs para cada uma das funcionalidades no lugar:
+
+- Opções depreciadas da configuração: Veja [o guia de migração da 0.26](/pt-br/guides/upgrade-to/v1/#new-configuration-api)
+- Tipos de atributo de script/style depreciados: Veja [o guia de migração da 0.26](/pt-br/guides/upgrade-to/v1/#new-default-script-behavior)
+
+### Removido: `image` do `astro:content` no esquema de coleções de conteúdo
+
+No Astro 2.x, a API de coleções de conteúdo depreciou a exportação `image` do `astro:content` utilizada no esquema de suas coleções de conteúdo.
+
+Astro 3.0 remove essa exportação completamente.
+
+#### O que devo fazer?
+
+Se você está utilizando o `image()` depreciado do `astro:content`, o remova, já que não ele não existe mais. Valide imagens através do [utilitário `image` de `schema`](/pt-br/guides/images/#atualize-esquemas-de-coleções-de-conteúdo) no lugar:
+
+ ```ts title="astro.config.mjs" del={1} ins={2} "({ image })"
+import { defineCollection, z, image } from "astro:content";
+import { defineCollection, z } from "astro:content";
+
+ defineCollection({
+   schema: ({ image }) =>
+     z.object({
+       image: image(),
+    }),
+});
+```
+
+### Removido: nomes de temas do Shiki pré-0.14
+
+No Astro v2.x, alguns nomes de tema do Shiki foram renomeados, mas os nomes originais foram mantidos por compatibilidade com versões anteriores.
+
+Astro v3.0 remove os nomes originais em favor dos nomes de tema renomeados.
+
+#### O que devo fazer?
+
+Se o seu projeto utiliza qualquer um dos temas abaixo, os renomeie para seu nome atualizado:
+
+- `material-darker` -> `material-theme-darker`
+- `material-default` -> `material-theme`
+- `material-lighter` -> `material-theme-lighter`
+- `material-ocean` -> `material-theme-ocean`
+- `material-palenight` -> `material-theme-palenight`
+
+### Removido: Funcionalidades do `class:list`
+
+No Astro v2.x, a [diretiva `class:list`](/pt-br/reference/directives-reference/#classlist) utilizava uma implementação customizada, inspirada por [`clsx`](https://github.com/lukeed/clsx) com algumas funcionalidades extras como desduplicação e suporte para `Set`.
+
+Astro v3.0 agora utiliza `clsx` diretamente para `class:list`, que não suporta desduplicação ou valores `Set`.
```

Comment:
```suggestion
Astro v3.0 agora utiliza `clsx` diretamente para `class:list`, que não suporta duplicação ou valores `Set`.
```

Reply:
This one is actually right! It looks like a typo for `duplicação` but the original says `deduplication`, which is translated as `desduplicação` see:
https://github.com/withastro/docs/pull/4166/files#diff-98423fea56caf403fbdc68882f98104aeee89ca2fdfab857e9f2d73c74c2c90aR225

Reply:
You have a good point! 😄